package.json

```diff
@@ -74,66 +74,12 @@
 		"yo": "^3.1.1"
 	},
 	"dependencies": {
-		"@ember/optional-features": "^2.0.0",
-		"@ember/test-helpers": "^2.1.4",
-		"@esri/arcgis-rest-auth": "^3.1.1",
-		"@esri/arcgis-rest-feature-layer": "^3.1.1",
-		"@esri/arcgis-rest-portal": "^3.1.2",
-		"@esri/arcgis-rest-request": "^3.1.1",
-		"@esri/arcgis-rest-service-admin": "^3.1.1",
-		"@esri/arcgis-rest-types": "^3.1.1",
-		"@esri/hub-types": "^6.10.0",
-		"@glimmer/component": "^1.0.3",
-		"@glimmer/tracking": "^1.0.3",
-		"@types/adlib": "^3.0.1",
-		"@types/base-64": "^0.1.3",
-		"@types/faker": "^5.1.5",
-		"adlib": "^3.0.7",
-		"babel-eslint": "^10.1.0",
-		"base-64": "^1.0.0",
-		"blob": "0.0.4",
-		"broccoli-asset-rev": "^3.0.0",
-		"cross-fetch": "^3.0.6",
-		"ember-auto-import": "^1.11.2",
-		"ember-cli": "~3.24.0",
-		"ember-cli-app-version": "^4.0.0",
-		"ember-cli-babel": "^7.23.0",
-		"ember-cli-dependency-checker": "^3.2.0",
-		"ember-cli-htmlbars": "^5.3.1",
-		"ember-cli-inject-live-reload": "^2.0.2",
-		"ember-cli-sri": "^2.1.1",
-		"ember-cli-terser": "^4.0.1",
-		"ember-data": "~3.24.0",
-		"ember-export-application-global": "^2.0.1",
-		"ember-fetch": "^8.0.2",
-		"ember-load-initializers": "^2.1.2",
-		"ember-maybe-import-regenerator": "^0.1.6",
-		"ember-page-title": "^6.0.3",
-		"ember-qunit": "^5.1.1",
-		"ember-resolver": "^8.0.2",
-		"ember-source": "~3.24.0",
```

Comment:
@dbouwman I don't know how all these dependencies (many of them ember???) got into the package root, but they should not be here. I restored the root dependences to what they were before.